docs/components/input-controller/_index.md

```diff
@@ -196,48 +196,44 @@ For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/c
 // Define a function that handles the controller.
```

Comment:
I have a few comments in this file; do you mind if I send you a revised version of the file offline with some explanation on what I changed?